BookPlayer/Player/PlayerManager.swift

```diff
@@ -54,33 +52,42 @@ class PlayerManager: NSObject {
             self.boostVolume = UserDefaults.standard.bool(forKey: Constants.UserDefaults.boostVolumeEnabled.rawValue)
 
             // Update UI on main thread
-            DispatchQueue.main.async(execute: {
-                // Set book metadata for lockscreen and control center
-                self.nowPlayingInfo = [
-                    MPMediaItemPropertyTitle: book.title,
-                    MPMediaItemPropertyArtist: book.author,
-                    MPMediaItemPropertyPlaybackDuration: audioplayer.duration
-                ]
-
-                self.nowPlayingInfo[MPMediaItemPropertyArtwork] = MPMediaItemArtwork(
-                    boundsSize: book.artwork.size,
-                    requestHandler: { (_) -> UIImage in
-                        return book.artwork
-                })
-
-                MPNowPlayingInfoCenter.default().nowPlayingInfo = self.nowPlayingInfo
-
-                if book.currentTime > 0.0 {
-                    self.jumpTo(book.currentTime)
+            DispatchQueue.main.async(
+                execute: {
+                    // Set book metadata for lockscreen and control center
+                    self.nowPlayingInfo = [
+                        MPMediaItemPropertyTitle: book.title,
+                        MPMediaItemPropertyArtist: book.author,
+                        MPMediaItemPropertyPlaybackDuration: audioplayer.duration,
+                        MPNowPlayingInfoPropertyDefaultPlaybackRate: self.speed,
+                        MPNowPlayingInfoPropertyPlaybackProgress: audioplayer.currentTime / audioplayer.duration,
+                    ]
+
+                    self.nowPlayingInfo[MPMediaItemPropertyArtwork] = MPMediaItemArtwork(
+                        boundsSize: book.artwork.size,
+                        requestHandler: { (_) -> UIImage in
+                            book.artwork
+                        }
+                    )
+
+                    MPNowPlayingInfoCenter.default().nowPlayingInfo = self.nowPlayingInfo
+
+                    if book.currentTime > 0.0 {
+                        self.jumpTo(book.currentTime)
+                    }
+
+                    // Set speed for player
+                    audioplayer.rate = self.speed
+
+                    NotificationCenter.default.post(name: .bookReady, object: nil, userInfo: ["book": book])
+
+                    if #available(iOS 11.0, *) {
+                        MPNowPlayingInfoCenter.default().playbackState = self.isPlaying ? MPNowPlayingPlaybackState.playing : MPNowPlayingPlaybackState.paused
```

Comment:
is this needed? only asking since the [doc](https://developer.apple.com/documentation/mediaplayer/mpnowplayinginfocenter/2588243-playbackstate) states that this only applies for MacOS 🤔